library/rabbitmq

```diff
@@ -7,7 +7,7 @@ Builder: buildkit
 
 Tags: 3.13.0-beta.6, 3.13-rc
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: ced055dcc013310157788d8e5a710f231bb7cdba
+GitCommit: b2387a802850aa877b8feefdd679e12536072db9
```

Comment:
These should be pointing at https://github.com/docker-library/rabbitmq/commit/6e5870068747ebef1d00134c52bd9a8e41cb2b4b as that is what `generate-stackbrew-library.sh` will end up with.